br/pkg/restore/systable_restore.go

```diff
@@ -96,6 +101,16 @@ func (rc *Client) RestoreSystemSchemas(ctx context.Context, f filter.Filter) {
 	tablesRestored := make([]string, 0, len(originDatabase.Tables))
 	for _, table := range originDatabase.Tables {
 		tableName := table.Info.Name
+		// --with-sys-table must be specified for recoverable system tables.
+		// One drawback of this filtering algorithm is that if a system table is not in both the unrecoverable and recoverable table list,
+		// the system table can be recovered  using only `--filter` flag
+		if !rc.withSysTable && isRecoverableSysTable(tableName.L) {
```

Comment:
this list only contains privileges table, there're other systables

Reply:
other systables are in `unRecoverableTable` list, is it ok?